cogs/r5rs.scm

```diff
@@ -417,8 +422,8 @@
 (check-equal? "case-insensitive string >=, true" #t (string-ci>=? "aa" "A"))
 (check-equal? "case-insensitive string >=, same string" #t (string-ci>=? "a" "A"))
 
-(skip-compile (check-equal #t (string=? "a" (make-string 1 #\a)))
-              (check-equal #f (string=? "a" (make-string 1 #\b))))
+(check-equal #t (string=? "a" (make-string 1 #\a)))
+(check-equal #f (string=? "a" (make-string 1 #\b)))
```

Comment:
These will need to be changed to be `check-equal? ` followed by a description of the test, see the other test cases for an example

Reply:
I am thinking it's better to remove these two additional test cases that I have added, since your first test case already checks for this scenario

Reply:
if possible, could we leave them in? It makes sure the result _isn't_ `string=?` when given a different seed character

Reply:
I have updated the PR